app/presenters/speedy_af/proxy/media_object.rb

```diff
@@ -27,6 +27,7 @@ def initialize(solr_document, instance_defaults = {})
     end
     # Handle this case here until a better fix can be found for multiple solr fields which don't have a model property
     @attrs[:section_id] = solr_document["section_id_ssim"]
+    @attrs[:section_ids] = solr_document["section_id_ssim"]
```

Comment:
I think they do because adding `@attrs[:section_ids]` allows this presenter to proxy the new `MediaObject#section_ids` method.  Alternatively this presenter could delegate `section_ids` to `section_id` or implement a new method which returns `section_id`.  Do you think either of those would make more sense?  I didn't want to remove `section_id` from the attributes because we had already added it so I know it is used elsewhere although I can't remember exactly where.

Reply:
Thinking on it more, it is probably best to just leave it like this. I do like the idea of delegating it but the current format is likely clearer at a glance.